### ISSUE TITLE
Could we bring back some styling decisions?

### ISSUE THREAD

**Issue by SvMak** [webjars/webjars#2105]
- indents between blocks are needed for better perception of information on the pages of the website;
- the section with features can be placed below the popular webjars section, because the table has a clear example of what advantages the use of the project gives;
- limitations of the table height allow to maintain the proportions of the page and scrolling allows to place more content

As an example, you can check the main page of the [zio project](https://zio.dev), I did its restyling.

**Comment by jamesward** [comment]
Thanks for the feedback :)

1. Yeah, I can increase indents
2. You are right that some of the features are conveyed by the table, so I propose keeping it at the top (so new users don't need to scroll to get a better understanding of what WebJars are), but we can probably reduce the feature list to 2x2 removing things that are duplicative of the table
3. With the table at the bottom of the page, I think it's ok to no have embedded scrolling. Can you give more details on what you don't like about the single-scrollbar?

**Comment by SvMak** [comment]
Before:

![Screenshot from 2024-11-04 12-15-44](https://github.com/user-attachments/assets/96c92bbd-2c01-4979-9b2f-fef4965c1db4)

![Screenshot from 2024-11-04 12-19-57](https://github.com/user-attachments/assets/5ada95b0-cd72-4d63-b596-b5e5df7356d2)


After:

![Screenshot from 2024-11-04 12-21-18](https://github.com/user-attachments/assets/498da8ad-1baa-4c16-a209-a3b6716eb950)

![Screenshot from 2024-11-04 12-20-56](https://github.com/user-attachments/assets/2535a193-d02d-4daf-acab-526b5c2345c9)

![Screenshot from 2024-11-04 12-22-23](https://github.com/user-attachments/assets/c47b62e5-8bfd-4b48-89ef-ed908cc4ab57)

Mobile view is broken now (((

Basically people make a list of improvements and I make these corrections. Maybe you didn't need to rush and we could smoothly fix everything you didn't like?


> 3\. With the table at the bottom of the page, I think it's ok to no have embedded scrolling. Can you give more details on what you don't like about the single-scrollbar?

I used embedded scrolling only in the context of dynamic table in top position of the page to avoid layout shifting issues.

**Comment by SvMak** [comment]
I wanted to thank you for this wonderful project!

Sorry for the offtopic.
For me, your project is not a random contribution.

Every project I contribute to, is important to me:
- SDKMAN! - best SDK manager for JVM ecosystem with responsive people;
- JHipster - battery-included project generator for Spring, Quarkus, Micronaut and many more.
Matt Raible, Deepu K Sasidharan, Baruch Sadogursky(He is not a participant in the project, but I became familiar with his speeches at conferences during the same period) - these guys have inspired me for many years)));
- DevJava - a wonderful project for learning the Java language, it's a sadly that the guys didn't give their feedback on the restyling;
- Njord - ORM for Rust language, a young project with great people;
- Zio - super set for creating various services on Scala;
- WebJars - a great project that helps connect a lot of frontend libraries to a project, I don't use it often, but it's no less important.



> I didn't expect you to be so helpful. :) Usually people drop changes, then disappear and I have to be the future maintainer. So I'm grateful that you want to be more involved than most! In the future I'll use PRs so we can collaborate on UI changes together.

sounds good



> Having things work on mobile is something we should strive for, but it isn't super critical for me as most people searching for WebJars are likely on a computer where they are working on code.

yes, but I think we can make the website usable on all devices)))



It would be great if you described your vision of the main page, and I would try to fix the appearance so that it looks good on all devices.

**Comment by SvMak** [comment]
Could you please check this: PR #2108
I'm used list of features items from your PR

**Comment by jamesward** [comment]
Thanks! Sooo grateful to have you helping out. Are there any other home page design decisions we should discuss?  I think it is looking and working great.  :)

**Comment by SvMak** [comment]
Thanks a lot for collaboration, now that's looking good ))
